mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -113,10 +117,34 @@ public static synchronized MapboxTelemetry getInstance() {
    * @param accessToken    The accessToken associated with the application
    * @param locationEngine Initialize telemetry with a custom location engine
    */
-  public void initialize(@NonNull Context context, @NonNull String accessToken,
-                         @NonNull String userAgent, @NonNull LocationEngine locationEngine) {
+  public void initialize(@NonNull Context context, @NonNull String accessToken, @NonNull String userAgent,
+                         @NonNull LocationEngine locationEngine) {
     this.locationEngine = locationEngine;
+
+    initialize(context, accessToken, userAgent);
+  }
+
+  /**
+   * Initialize MapboxTelemetry - with sdkIdentifier + sdkVersion
+   *
+   * @param sdkIdentifier  Identifies which sdk is sending the event
+   * @param sdkVersion version of the sdk sending the event
+   */
+
+  public void initialize(@NonNull Context context, @NonNull String accessToken, @NonNull String userAgent,
+                         String sdkIdentifier, String sdkVersion) {
```

Comment:
We should add `@NonNull` annotation to `sdkIdentifier` and `sdkVersion` to reflect that those should be non `null`.